pandas/tests/extension/test_boolean.py

```diff
@@ -397,3 +397,21 @@ class TestParsing(base.BaseParsingTests):
 
 class Test2DCompat(base.Dim2CompatTests):
     pass
+
+
+def test_from_arrow(dtype):
```

Comment:
you might need to use a min version here in the skip and use the decorator version like
`td.skip_if_no("pyarrow", min_version="1.0.0")`

Reply:
Can you also move the test to `/tests/arrays/boolean/` ?

Reply:
Or more specifically, we have tests for arrow compat in `tests/arrays/masked/test_arrow_compat.py`. I would add it there (or first check if it's not actually already covered)